core/src/main/java/com/datastrato/gravitino/catalog/CatalogOperationDispatcher.java

```diff
@@ -68,15 +37,15 @@
  * A catalog operation dispatcher that dispatches the catalog operations to the underlying catalog
  * implementation.
  */
-public class CatalogOperationDispatcher implements TableCatalog, FilesetCatalog, SupportsSchemas {
+public abstract class CatalogOperationDispatcher {
```

Comment:
Can we rename this class to `OperationDispatcher`?

Reply:
done